app/views/posts/_write_reply.haml

```diff
@@ -4,9 +4,7 @@
     .view-button#rtf{class: ('selected' if (params[:editor_mode] || current_user.default_editor) == 'rtf')} Rich Text
     .view-button#html{class: ('selected' if (params[:editor_mode] || current_user.default_editor) == 'html')} HTML
     %br
-    - if unread_warning
-      = unread_warning
-      = hidden_field_tag :unread_warned, true
+    = hidden_field_tag :last_seen, @last_seen_id
```

Comment:
This no longer warns the user before submitting the reply that there are unread replies. (If someone spends a while composing a tag only to find they failed to notice there was a new page in the thread, and that they were halfway through their coauthor's series of tags, that's a problem that would previously have been solved by the warning.)